staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -42,12 +42,18 @@ type GenericWebhook struct {
 	initialBackoff time.Duration
 }
 
-// NewGenericWebhook creates a new GenericWebhook from the provided kubeconfig file.
-func NewGenericWebhook(registry *registered.APIRegistrationManager, codecFactory serializer.CodecFactory, kubeConfigFile string, groupVersions []schema.GroupVersion, initialBackoff time.Duration) (*GenericWebhook, error) {
-	return newGenericWebhook(registry, codecFactory, kubeConfigFile, groupVersions, initialBackoff, defaultRequestTimeout)
-}
-
-func newGenericWebhook(registry *registered.APIRegistrationManager, codecFactory serializer.CodecFactory, kubeConfigFile string, groupVersions []schema.GroupVersion, initialBackoff, requestTimeout time.Duration) (*GenericWebhook, error) {
+// LoadClientConfig loads a kubeconfig-formatted file at filesystem path
```

Comment:
let's split unrelated util/webhook and authorizer/webhook changes out of this PR (and I think just the doc changes are relevant now)